CONTRIBUTING.md

```diff
@@ -8,6 +8,13 @@ When you submit a pull request, a CLA bot will automatically determine whether y
 a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
 provided by the bot. You will only need to do this once across all repos using our CLA.
 
```

Comment:
Why did you delete the below?  I thought that was the legal-approved text that we need to keep in there.  I like your additions but I think we need those links that were deleted as well.

Reply:
@chelseacarter29  I added this back with commit e13ac4bc67a363b7bea56ed015c38eb5eae4c34e.  Let me know if this is not ok.